deploy/future

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+# Don't run this with -e, grep exits with 1 when it finds no results
+
+function check_py () {
+# This is a bit janky. If you know how to get the output
+# to go on separate lines, please tell me.
+    results=`git --no-pager grep $1|grep "\.py"|grep -v "future-ok"`
```

Comment:
I'd use some variation on the more standard `# noqa` instead, just for consistency (see e.g. [flake8](https://pypi.python.org/pypi/flake8)).

It looks like you can just do `git grep $1 -- '*.py'` instead of grepping the output to get the extension.  And you shouldn't need `--no-pager`, I think it should autodetect that if the output is not connected to a terminal.